s/soap-common/soap-common_ubi8.4.sh

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : soap-common
+# Version       : 1.0.0-SNAPSHOT
+# Source repo   : https://github.com/reficio/soap-ws
+# Tested on     : ubi: 8.4
+# Script License: Apache License 2.0
+# Maintainer's	: Sapna Shukla <Sapna.Shukla@ibm.com> 
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables.
+PACKAGE_NAME=soap-ws/soap-common
+PACKAGE_URL=https://github.com/reficio/soap-ws
+
```

Comment:
Also use variable for package version.
`PACKAGE_VERSION=${1:-1.0.0-SNAPSHOT}`
Check out the PACKAGE_VERSION after `git clone`

Reply:
For the given package it has just a master branch
url:  https://github.com/reficio/soap-ws

On making the suggested change, it gives:
error: pathspec '1.0.0-SNAPSHOT' did not match any file(s) known to git

Can however make  this PACKAGE_VERSION=master
Added the commit with that change

Reply:
In that case, can you take the commit-id (instead of master) as the version and check-out while building the code.

Reply:
Added commit ID